src/Illuminate/Session/SessionManager.php

```diff
@@ -103,7 +105,18 @@ protected function createApcDriver()
 	 */
 	protected function createMemcachedDriver()
 	{
-		return $this->createCacheBased('memcached');
+		$store = $this->app['config']['session.memcached_store'];
+
+		// Verify store uses the memcached driver
+		$repository = $this->app['cache']->store($store);
+		if (!($repository->getStore() instanceof MemcachedStore))

```

Comment:
Could you let me know what CS needs fixing here? Be useful to know for future too.

Looking at PSR-1 and the [Laravel standards](http://laravel.com/docs/master/contributions) I can't immediately see what's wrong.

Thanks